lib/ansible/modules/network/ordnance/ordnance_config.py

```diff
@@ -0,0 +1,400 @@
+        for cmd in ['config terminal', key, value, delimiter, 'end']:
+            cmd += '\r'
+            module.connection.shell.shell.sendall(cmd)
+        time.sleep(1)
+        module.connection.shell.receive()
+
+def get_config(module, result):
+    contents = module.params['config']
+    if not contents:
+        defaults = module.params['defaults']
+        contents = module.config.get_config(include_defaults=defaults)
+
+    contents, banners = extract_banners(contents)
+    return NetworkConfig(indent=1, contents=contents), banners
+
+def get_candidate(module):
+    candidate = NetworkConfig(indent=1)
+    banners = {}
+
+    if module.params['src']:
+        src, banners = extract_banners(module.params['src'])
+        candidate.load(src)
+
+    elif module.params['lines']:
+        parents = module.params['parents'] or list()
+        candidate.add(module.params['lines'], parents=parents)
+
+    return candidate, banners
+
+def run(module, result):
+    match = module.params['match']
+    replace = module.params['replace']
+    path = module.params['parents']
+
+    candidate, want_banners = get_candidate(module)
+
+    if match != 'none':
+        config, have_banners = get_config(module, result)
+        path = module.params['parents']
+        configobjs = candidate.difference(config, path=path,match=match,
+                                          replace=replace)
+    else:
+        configobjs = candidate.items
+        have_banners = {}
+
+    banners = diff_banners(want_banners, have_banners)
+
+    if configobjs or banners:
+        commands = dumps(configobjs, 'commands').split('\n')
+
+        if module.params['lines']:
+            if module.params['before']:
+                commands[:0] = module.params['before']
+
+            if module.params['after']:
+                commands.extend(module.params['after'])
+
+        result['updates'] = commands
+        result['banners'] = banners
+
+        # send the configuration commands to the device and merge
+        # them with the current running config
+        if not module.check_mode:
+            if commands:
+                module.config(commands)
+            if banners:
+                load_banners(module, banners)
+
+        result['changed'] = True
+
+    if module.params['save']:
+        if not module.check_mode:
+            module.config.save_config()
+        result['changed'] = True
+
+def main():
+    """ main entry point for module execution
+    """
+
+    argument_spec = dict(
+        src=dict(type='path'),
+
+        lines=dict(aliases=['commands'], type='list'),
+        parents=dict(type='list'),
+
+        before=dict(type='list'),
+        after=dict(type='list'),
+
+        match=dict(default='line', choices=['line', 'strict', 'exact', 'none']),
+        replace=dict(default='line', choices=['line', 'block']),
+        multiline_delimiter=dict(default='@'),
+
+        # this argument is deprecated in favor of setting match: none
```

Comment:
So you can remove the following lines (and from docs)
```
# this argument is deprecated in favor of setting match: none
# it will be removed in a future version
force=dict(default=False, type='bool'),

```